INSTALL.md

```diff
@@ -582,6 +582,14 @@ alternative, you can use the language specific variables, `CFLAGS` and `CXXFLAGS
 Build only some minimal set of features.
 This is a developer option used internally for CI build tests of the project.
 
+### no-cached-fetch
+
+Never cache algorithms after they are fetched from a provider.  Normally, a
```

Comment:
Is 'after' correct or should it be 'when'

Reply:
The cache is used after fetching for future queries.  When works too.

Reply:
Changed to when